### ISSUE TITLE
No Issue, VMware Possible

### ISSUE THREAD

**Issue by Alveran81** [eq-3/occu#78]
Is it possible to get the HM-OCCU-SDK published to a VM inside ESXi 6.5 ?

**Comment by jp112sdl** [comment]
Sure, just install Debian in a virtual machine and use the subfolder for [Debian_Wheezy](https://github.com/eq-3/occu/tree/master/X86_32_Debian_Wheezy)

**Comment by Alveran81** [comment]
Yes of course. But how Adapter works? Is there any Chance with a USB Converter to get in Virtual Machine?

**Comment by jp112sdl** [comment]
I don't think so. You have to use the LAN-Gateway for communication with 'classic' HomeMatic devices.
For HmIP devices you need to connect the new HmIP USB Stick.

But in my opinion, that's not a useful construct.
